src/app/notifications/components/SearchFailure.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
```

Comment:
This probably is not must-have, but I'm more into using a structure like `components/SearchFailure/index.tsx` and then make a separate file with style for this component like `../styled.ts(x)`
It would probably nice to make this a functional component as this is recommended by the react team

Reply:
It was a functional component before but i had trouble testing the clean up function in useEffect, I will try again

Reply:
I'll accept without this change, so what works best for you